app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -57,11 +68,30 @@ protected void onCreate(Bundle savedInstanceState) {
         // Notify presenter to attach
         presenter.attach(this);
 
+        checkBoxEnableUrl.setOnCheckedChangeListener((buttonView, isChecked) -> {
+            if(isChecked) {
+                checkBoxEnableUrl.setTextColor(Color.parseColor("#000000"));
```

Comment:
As to shift this block to presenter, it doesn't contain any interaction with data model or anything else other than just view itself. Hence no point to shift this to presenter. AFAIK